frontend/src/components/Bundle/Bundle.js

```diff
@@ -148,9 +148,8 @@ class Bundle extends React.Component<
                             }
                         }.bind(this),
                     );
-                    $.when.apply($, fetchRequests).then(() => {
```

Comment:
I'm actually a bit confused about what this logic is actually doing. Why is it calling `$.when` with `fetchRequests` as the argument twice?

Reply:
https://medium.com/sungthecoder/making-multiple-ajax-calls-and-deciphering-when-apply-array-b35d1b4b1f50
So here we're doing multiple AJAX calls and only go on to the then command if all of them are resolved.

Reply:
Ah, I see. I think it's clearer to instead just do `Promise.all(...fetchRequests)` as that is native to JS and doesn't require jQuery.

I'm still confused as to why we also have `$.when(fetchRequests)` on line 154, though? That's what I meant about "calling $.when with fetchRequests as the argument twice".

Feel free to ping me on Slack so we can discuss this further.